ai-platform-unified/notebooks/notebook_template.ipynb

```diff
@@ -0,0 +1,626 @@
+{
```

Comment:
<p>I find not specifying the "gs://" here for BUCKET_NAME is prone to later typos, where you always have to add the prefix. So I just solve the problem and do it once:</p><p> </p><p>"[your-bucket-name]" -&gt; "gs://[your-bucket-name]"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/251/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Updated</p>